components/esm/cellref.cpp

```diff
@@ -19,10 +40,8 @@ void ESM::CellRef::loadId(ESMReader &esm, bool wideRefNum)
     if (esm.isNextSub ("NAM0"))
         esm.skipHSub();
 
-    if (wideRefNum)
-        esm.getHNT (mRefNum, "FRMR", 8);
-    else
-        esm.getHNT (mRefNum.mIndex, "FRMR");
+    if (!ignoreRefNum)
```

Comment:
I can't find any uses of the ignoreRefNum parameter. Is that a leftover of earlier code?